webextension/scripts/utils.js

```diff
@@ -176,7 +176,7 @@ function remove_wbm(url) {
 
 // Function to clean the URL if the user is on 'web.archive.org'
 function get_clean_url(url) {
-  if (url.includes('web.archive.org')) {
+  if (url && url.includes('web.archive.org')) {
```

Comment:
Thank you for adding `url &&` to the if-condition statement. I tried `get_clean_url(makeValidURL(event.target.innerHTML))` earlier but was getting `Cannot read property 'includes' of null` earlier, in case when `null` was returned :-)